charts/posthog/README.md

```diff
@@ -143,31 +221,21 @@ worker:
 
 plugins:
   replicacount: 2
-
-pgbouncer:
```

Comment:
Why?

Reply:
That's the default, so skip it makes things shorter/simpler